sqlmesh/core/snapshot/evaluator.py

```diff
@@ -1549,3 +1578,20 @@ def _intervals(snapshot: Snapshot, deployability_index: DeployabilityIndex) -> I
         if deployability_index.is_deployable(snapshot)
         else snapshot.dev_intervals
     )
+
+
+def _check_destructive_schema_change(
+    snapshot: Snapshot,
+    alter_expressions: t.List[exp.AlterTable],
+    allow_destructive_snapshots: t.Set[str],
+) -> None:
+    if snapshot.needs_destructive_check(
+        allow_destructive_snapshots, runtime_check=True
+    ) and snapshot.has_drop_alteration(alter_expressions):
+        warning_msg = f"RUN TIME CHECK: Plan results in a destructive change to forward-only table '{snapshot.name}'s schema."
```

Comment:
`RUN TIME CHECK` - where does this message prefix come from? Is this some new convention?